Makefile

```diff
@@ -19,9 +20,9 @@ install-python-requirements:
 install-python-requirements-dev: install-python-requirements
 	$(PYTHON) -m pip install -Ur requirements-dev.txt
 install-deps:
-	pkg install -q -y libucl py$(pyver)-ucl py$(pyver)-cython rsync python$(pyver) py$(pyver)-libzfs
+	pkg install -q -y libucl py$(pyver)-ucl py$(pyver)-setuptools py$(pyver)-wheel rsync python$(pyver) py$(pyver)-libzfs
```

Comment:
Why do we need setuptools and wheel?

Reply:
because it wasn't able to install dependencies without